cves/kernel/CVE-2014-4943.yml

```diff
@@ -167,8 +174,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    It is unlikely that this was discovered throw automation because it 
```

Comment:
"Through" instead of "throw" but possibly reword it to "discovered from automation"